transaction-status/src/parse_stake.rs

```diff
@@ -354,7 +354,7 @@ mod test {
         );
         let message = Message::new(&[instruction], None);
         assert_eq!(
-            parse_stake(&message.instructions[0], &keys[0..2]).unwrap(),
+            parse_stake(&message.instructions[0], &keys[0..3]).unwrap(),
```

Comment:
It would be nice to add clock sysvar to the decoded json, when included in the instruction. Perhaps timed with #14587

Reply:
Yep, that's what I was thinking.   Same for the web3.js decoder too

Reply:
web3.js decoder PR at https://github.com/solana-labs/solana/issues/14587